src/components/MatchBreakdownTable/breakdown2019.js

```diff
@@ -0,0 +1,351 @@
+import React from "react";
+import Close from "@material-ui/icons/Close";
+import CheckRounded from "@material-ui/icons/CheckRounded";
+import LensIcon from "@material-ui/icons/Lens";
+import TripOriginIcon from "@material-ui/icons/TripOrigin";
+import Tooltip from "@material-ui/core/Tooltip";
+import PropTypes from "prop-types";
+import BreakdownRow from "../BreakdownRow";
+
+const MatchBreakdown2019 = ({ match }) => {
+  match = match.toJS();
+  const redTeams = match.alliances.red.team_keys.map(teamKey =>
+    teamKey.replace("frc", "")
+  );
+  const blueTeams = match.alliances.blue.team_keys.map(teamKey =>
+    teamKey.replace("frc", "")
+  );
+  const redBreakdown = match.score_breakdown.red;
+  const blueBreakdown = match.score_breakdown.blue;
+  return (
+    <div>
+      <BreakdownRow
+        data={[
+          "Teams",
+          redTeams.map(team => (
+            <span key={team} style={{ display: "block" }}>
+              {team}
+            </span>
+          )),
+          blueTeams.map(team => (
+            <span key={team} style={{ display: "block" }}>
+              {team}
+            </span>
+          )),
+        ]}
+        vertical={true}
+        subtotal={true}
+      />
+
+      <BreakdownRow
+        data={[
+          "Robot 1 Sandstorm Bonus",
+          getSandstormBonusFor(redBreakdown, 1),
```

Comment:
This `get*For` pattern isn't very "React." I guess `getSandstormBonusFor` is fine since it's just a string, but for things like `getCargoShipDataFor` it's awkward and problematic. See below.

Reply:
Per the discussion in #dev-ios in Slack, can you change this to not pass the data as an array, but rather as separate props?